doc/userguide/rules/ssh-keywords.rst

```diff
@@ -6,6 +6,15 @@ Suricata has several rule keywords to match on different elements of SSH
 connections.
 
 
+Frames
+------
+
+The SSH parser supports the following frames:
+
+* ssh.record_hdr
+* ssh.record_data
+* ssh.record_pdu
```

Comment:
Can this get some more explanation and some examples too perhaps?

Reply:
Trying something